src/Raven.Studio/typescript/common/generalUtils.ts

```diff
@@ -505,6 +505,17 @@ class genUtils {
 
         return result;
     }
+    
+    static truncateDocumentId(documentId: string) {
+        const charsToShow = 12;
```

Comment:
add at the beggining: if (!documentId) { return null }

Reply:
done